helpers/QtiPackageExporter.php

```diff
@@ -20,40 +20,99 @@
 
 namespace oat\taoQtiTest\helpers;
 
-use League\Flysystem\Util;
-use oat\oatbox\service\ConfigurableService;
+use common_Exception;
+use common_exception_Error;
+use common_report_Report;
+use oat\oatbox\filesystem\File;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\tao\helpers\FileHelperService;
+use oat\tao\model\service\InjectionAwareService;
 use taoQtiTest_models_classes_export_TestExport22 as TestExporter;
-use core_kernel_classes_Resource as RdfResource;
-use tao_helpers_File as FileHelper;
 
-class QtiPackageExporter extends ConfigurableService
+class QtiPackageExporter extends InjectionAwareService
 {
+    public const SERVICE_ID = 'taoQtiTest/QtiPackageExporter';
+    public const QTI_PACKAGE_FILENAME = 'qti_test_export';
+
+    /** @var TestExporter */
+    private $testExporter;
+
+    /** @var FileSystemService */
+    private $fileSystemService;
+
+    /** @var FileHelperService */
+    private $fileHelperService;
+
+    public function __construct(
+        TestExporter $testExporter,
+        FileSystemService $fileSystemService,
+        FileHelperService $fileHelperService
```

Comment:
I think it would be better if the helper service (which should not exist by the way) would be used by the FileSystemService

Reply:
I need to clarify  why the [method to register local file system](https://github.com/oat-sa/generis/blob/develop/common/oatbox/filesystem/FileSystemService.php#L151) with flysystem abstraction is registered as deprecated. If we can use it, we won't need this helper at all.